### ISSUE TITLE
Has anyone tried installing workflow on amazon Lightsail?

### ISSUE THREAD

**Issue by helloravi** [deis/workflow#735]
I have tried looking up for examples of implementations of Deis on lightsail. I did not find any examples. Has anyone tried doing that?

**Comment by Joshua-Anderson** [comment]
Workflow requires a running kubernetes cluster. There is already a lot of tooling and scripts around spinning up an kubernetes cluster on ec2.

Lightsail is nice for a easy linux server, but ec2 will work better for the more complex needs of a kubernetes cluster, which requires several virtual machines networked together.

**Comment by bacongobbler** [comment]
Yep, as far as I know nobody's written up or wrote tooling around getting k8s running on amazon lightsail. That being said we'd be open to docs!

**Comment by helloravi** [comment]
@Joshua-Anderson 
I found the following on [aws documentation](https://aws.amazon.com/blogs/aws/amazon-lightsail-the-power-of-aws-the-simplicity-of-a-vps/) at the bottom:

All of the Lightsail instances within an account run within a “shadow” VPC that is not visible in the AWS Management Console. If the code that you are running on your Lightsail instances needs access to other AWS resources, you can set up VPC peering between the shadow VPC and another one in your account, and create the resources therein. Click on Account (top right), scroll down to Advanced features, and check VPC peering:

**Comment by helloravi** [comment]
Here is another stackoverflow question on Peering LIghtsail instances
http://serverfault.com/questions/818280/cant-establish-vpc-peering-connection-from-amazon-lightsail

**Comment by Cryptophobia** [comment]
This issue was moved to teamhephy/workflow#38
